### PR TITLE
Corrected some commands in the mesheryctl installation page

### LINKED ISSUE #3117: [Docs]Mesheryctl installation page need some correction
#### Current State

In the [mesheryctl installation](https://docs.meshery.io/installation/mesheryctl) page under the bash command section the command gives an unexpected error of not mentioning the platform [docker or kubernetes] and it does not have the installation guide for the Kubernetes user like it is provided in the [website](https://meshery.io/).

#### Desired State

In the mesheryctl installation page under the bash command section the platform must be mentioned and the installation step for the Kubernetes user must be provided

---

#### Contributor Resources

- [Meshery documentation site](https://docs.meshery.io/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
- [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by sayantan1413**
Signed-off-by: sayantan1413 <sayantanbose2001@gmail.com>

**Description**

Corrected some commands in the mesheryctl installation page and added the kubernetes installation steps.

This PR fixes #3117 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: docs/pages/installation/mesheryctl.md [hunk 1]
```diff
@@ -14,10 +14,20 @@ Meshery's command line client is `mesheryctl`. To install `mesheryctl` on your s
 ## Bash
 
 **Install** and **Upgrade**
+<h6>Install on docker</h6>
```
Comment (leecalcote) [inline docs/pages/installation/mesheryctl.md:17]: Capitalize Docker

File: docs/pages/installation/mesheryctl.md [hunk 2]
```diff
@@ -62,6 +85,44 @@ Removing: /Users/lee/Library/Caches/Homebrew/mesheryctl--0.3.2.zip... (3.9MB)
...
+Not a Helm user? Use the Meshery manifests directly.
+
+<h6>Using Manifests</h6>
+
+<pre class="codeblock-pre">
+<div class="codeblock">
+ <div class="clipboardjs">
+ $ git clone https://github.com/layer5io/meshery.git; cd meshery
+ $ kubectl create ns meshery
+ $ kubectl -n meshery apply -f install/deployment_yamls/k8s
```
Comment (leecalcote) [inline docs/pages/installation/mesheryctl.md:122]: Will these "$"s be captured into the user's clipboard? If so, they'll need removed.

Comment (sayantan1413) [reply docs/pages/installation/mesheryctl.md:122]: No the "$"s don't get captured in the user's clipboard.

File: docs/pages/installation/mesheryctl.md (hunk 1 above)
Comment (sayantan1413) [reply docs/pages/installation/mesheryctl.md:17]: Okay changing it.

File: docs/pages/installation/mesheryctl.md (hunk 2 above)
Comment (leecalcote) [reply docs/pages/installation/mesheryctl.md:122]: Cool. In this case, let's be consistent with their use and incorporate the "$"s in the clipboardjs blocks above.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: docs/pages/installation/mesheryctl.md [hunk 3]
```diff
@@ -14,10 +14,20 @@ Meshery's command line client is `mesheryctl`. To install `mesheryctl` on your s
 ## Bash
 
 **Install** and **Upgrade**
+<h6>Install on Docker</h6>
+ <pre class="codeblock-pre">
+ <div class="codeblock">
+ <div class="clipboardjs">
+ $ curl -L https://git.io/meshery | PLATFORM=docker bash -
```
Comment (leecalcote) [inline docs/pages/installation/mesheryctl.md:21]:
```suggestion
  $ curl -L https://git.io/meshery | PLATFORM=docker bash -
```

File: docs/pages/installation/mesheryctl.md [hunk 4]
```diff
@@ -14,10 +14,20 @@ Meshery's command line client is `mesheryctl`. To install `mesheryctl` on your s
...
+ </pre>
+ 
 
+<h6>Install on Kubernetes</h6>
  <pre class="codeblock-pre">
- <div class="codeblock"><div class="clipboardjs">
- curl -L https://git.io/meshery | bash -
+ <div class="codeblock">
+ <div class="clipboardjs">
+ $ curl -L https://git.io/meshery | PLATFORM=kubernetes bash -
```
Comment (leecalcote) [inline docs/pages/installation/mesheryctl.md:30]:
```suggestion
  $ curl -L https://git.io/meshery | PLATFORM=kubernetes bash -
```

**Review by leecalcote — APPROVED** [review]

**Pull request merged**